BlockScience/cadCAD
# merge setup.py update

(fixes #70)

This is an update to setup.py to include the `install_requires` keyword, so that pip automatically installs the dependencies when cadCAD is installed.

How to test this update:
1. uninstall cadCAD (`pip uninstall cadCAD`)
1. uninstall one of its dependencies (eg. `pip uninstall pathos`)
1. install cadCAD from the github fork/branch (`pip install git+https://github.com/markusbkoch/cadCAD.git@markusbkoch-patch-1`)

This should result in pathos and cadCAD being installed.
